app/javascript/src/component_dialog_form.js

```diff
@@ -104,7 +105,7 @@ class FormDialog {
   }
 
   get state() {
-    return this.$node.attr(STATE_ATTR);;
+    return this.$container.attr(FormDialog.STATE_ATTR);
```

Comment:
Is there a reason why you're reading the state from the html element rather than simply returning the value of the class private property? Is it because that's how the underlying jQuery UI dialog does it? 

Maybe because `#state` doesn't appear to get set/changed after the constructor?

If we're not using the `#state` property and just using the attribute, then we should probably remove `#state` to avoid confusion.

Reply:
See earlier comments about previous efforts and #state no actually being used. 

Also, at risk of repeating my first reply, I've simply replaced everything with a dialog.isOpen() wrapper to the (discovered) already existing dialog.$node.dialog("isOpen") jQueryUI Dialog provided route. 

Other than simplicity of access, I also replaced state() with isOpen() to make this component more consistent with your recent Expander change.